frontend/src/pages/distributedWorkloads/global/GlobalDistributedWorkloads.tsx

```diff
@@ -7,6 +7,7 @@ import { DistributedWorkloadsContextProvider } from '~/concepts/distributedWorkl
 import { DistributedWorkloadsTabConfig } from '~/pages/distributedWorkloads/global/useDistributedWorkloadsTabs';
 import DistributedWorkloadsNoProjects from '~/pages/distributedWorkloads/global/DistributedWorkloadsNoProjects';
 import GlobalDistributedWorkloadsTabs from '~/pages/distributedWorkloads/global/GlobalDistributedWorkloadsTabs';
+import ModelServingProjectSelection from '~/pages/modelServing/screens/global/ModelServingProjectSelection';
```

Comment:
We shouldn't use anything from other `~/pages/*` subdirectories within here (a rule I know is broken elsewhere, but @andrewballantyne requested we stick to that for new code). Can we maybe move `ModelServingProjectSelection` into `~/concepts/projects/` and rename it something like `PageHeaderProjectSelection`, or just make our own `DistributedWorkloadsProjectSelection`?

Reply:
Ah man, this is getting fixed in the DS Projects rework PR... 🤦 We have two types of this component goofy like...

Maybe just update yours to match what he will do for the joys of the magic here? #2572 

https://github.com/opendatahub-io/odh-dashboard/blob/ad564f126f40e4372f76295c2a777134c43f184a/frontend/src/concepts/projects/ProjectSelector.tsx

Reply:
Ah, glad I tagged you then. Maybe we just hold off on this until after #2572 is merged

Reply:
You're welcome to take those exact changes over to your PR, I doubt they will change. Don't feel blocked by this effort.